README.md

```diff
@@ -24,72 +42,161 @@ Does:
 4. Store default filter value, which selects DHCPACK and DHCPREQUESTS in a DHCP mask.
 5. Store those values in DHCP-Forwarder.toml in the working directory.
 
+Note: Do not select a Wireless device, it will not work.
 
-dhcp-forwarder.exe
-===========
-A quick go tool to mirror a UDP DHCP packet capture based on a BPF and forward it to a configured host and port.
-Support a configuration file in DHCP-Forwarder in TOML format, that needs to be present in the same directory as the binary.
+The DHCP-Forwarder service needs to be restarted:
 
-DHCP-Forwarder.toml file is expected to be present in the working directory, which is generated from dhcp-forwarder-config-generator.exe
+1. After a configuration change
+2. When the server goes to sleep and resumes
 
 
-Native Compilation under x64:
-===========
-You will need:
+The installer
+-------------
+The installer will:
```

Comment:
start each bullet with a lower-case letter, only the last line should have a trailing dot.